src/main/java/edu/harvard/iq/dataverse/engine/command/impl/RevokeRoleCommand.java

```diff
@@ -38,5 +39,9 @@ protected void executeImpl(CommandContext ctxt) throws CommandException {
         return Collections.singletonMap("",
                 toBeRevoked.getDefinitionPoint() instanceof Dataverse ? Collections.singleton(Permission.ManageDataversePermissions)
                 : Collections.singleton(Permission.ManageDatasetPermissions));
-    }	
+    }
+    
+    @Override public String describe() { 
+	    return toBeRevoked.getAssigneeIdentifier() + " has had the role: " + toBeRevoked.getRole() + " REVOKED on " + toBeRevoked.getDefinitionPoint().accept(DvObject.NameIdPrinter); 
```

Comment:
I assume...

```
" REVOKED on " + toBeRevoked.getDefinitionPoint().accept(DvObject.NameIdPrinter); 
```

.. is used instead of something shorter like...

```
" REVOKED on " + toBeRevoked.getDefinitionPoint().getDisplayName(); 
```

... because we want the database ID's as well as the name in the output.

Reply:
I'm assuming it was done this way, as that is what is in AssignRoleCommand:

https://github.com/IQSS/dataverse/commit/7cfc8bab11e24daca562f2a3ae16cf737e94487e#diff-634bf68267b3070eb7ee7afe2a4e2fde1e282e73d3ad4f553bfbe4c458a00794

(which was originally added by... @pdurbin :) )

Reply:
![homer](https://user-images.githubusercontent.com/21006/140338768-e62bc556-2175-420e-88f5-1a4959c4199c.jpeg)

Reply:
I followed the AssignRoleCommand way of writing it as I assumed it was approved in a code review so did the same.

If there's a better/preferred way, I can change it.

Reply:
Nope. Seems fine to me. Consistent. Thanks.

Reply:
I will create another issue for the revokeAll() command.